code/game/objects/structures/fence.dm

```diff
@@ -89,7 +90,10 @@
 	if(shock(user, 100))
 		return
 	if(!cuttable)
-		to_chat(user, "<span class='warning'>This section of the fence can't be cut!</span>")
+		if(W.use_tool(src, user, FULL_CUT_TIME * W.toolspeed, volume = W.tool_volume))
```

Comment:
Might want to give feedback when starting in the form of text. Probably a `visible_message` for others as well